docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+# App publishing guide
+
+After you've completed developing and testing your app's package for Sandstorm, you can publish it in the [Sandstorm App Market](https://apps.sandstorm.io/).
+
+## Verify your identity
+
+### Sign up with [Keybase.io](https://keybase.io)
+
+Currently, Keybase is invite-only. If you need an invite, you can contact [Jade](mailto:jade@sandstorm.io) or [Kenton](mailto:kenton@sandstorm.io). You should connect some of your public identites with your Keybase account, like Twitter and Github.
+
+Sandstorm app authors are verified using a PGP key linked to Keybase. You should get the [prerequisites](https://keybase.io/docs/command_line/prerequisites) and follow [their directions](https://keybase.io/docs/command_line/installation) to get their software set up.
+
+### Sign your app
+
+In order to verify that you are the author of the app in question, you need to sign the following ASCII statement: `I am the author of the Sandstorm.io app with the following ID: <app-id>`, where `<app-id>` is the one from your `sandstorm-pkgdef.capnp` file.
```

Comment:
To be a little bit pedantic, you are not signing your _app_ this way -- the app is already signed by your key for the app in ~/.sandstorm-keyring.

You might say this is "Adding a note to your Sandstorm package linking it to your Keybase key."

Technically speaking, you are signing an assertion (with your GPG key) that you are the holder of this (Sandstorm) app key, and then by doing `spk pack` you sign _that_ with your Sandstorm app key, since all packages are signed by a Sandstorm app key.

I'm open to any explanation you want, but I'd prefer that it be accurate, even if also vague. (-: